drivers/video/video_mcux_csi.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2019, Linaro Limited
+ * Copyright 2024 NXP
```

Comment:
This can be in the first commit where we touch the CSI file in this PR

Reply:
In fact, we touched the CSI driver several times in the past and I explictly added the NXP Copyright here when seeing that the driver is changed enough by NXP (although I don't know how much change is "enough" to add a copyright).